home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/contacts/ContactApiIT.java

```diff
@@ -414,7 +414,7 @@ void passNullIdContactWhenDeleteAnyContactTest() throws ApiException {
         assertThatExceptionOfType(ApiException.class)
             .isThrownBy(() -> contactApi
                 .deleteContactOnUserWithHttpInfo(expectedUser.getId(), null))
-            .withMessageContaining("Missing the required parameter 'contactId' when calling deleteContactOnUser");
+            .withMessageContaining("Missing the required parameter 'id' when calling deleteContactOnUser");
```

Comment:
Isn't it an error that client throws? I mean without any real API call?

Reply:
Yes, it is but this is not the essence of the task. A separate task will be created to solve this problem.